rules/scala/private/doc.bzl

```diff
@@ -24,7 +24,7 @@ def scaladoc_implementation(ctx):
     html = ctx.actions.declare_directory("html")
     tmp = ctx.actions.declare_directory("tmp")
 
-    classpath = depset(transitive = [dep[JavaInfo].transitive_compile_time_deps for dep in ctx.attr.deps])
+    classpath = depset(transitive = [dep[JavaInfo].transitive_deps for dep in ctx.attr.deps])
```

Comment:
I think this is right, but I'm not 100%. @pauldraper do you know if this is correct?

Reply:
This is correct, for the current versions of Bazel.

Reply:
Thanks!